awscli/customizations/scalarparse.py

```diff
@@ -36,13 +39,29 @@ def identity(x):
     return x
 
 
+def iso_format(value):
+    return parse_timestamp(value).isoformat()
+
+
+def choose_default_parsers(timestamp_format):
```

Comment:
Does it? Looking at the implementation: https://github.com/boto/botocore/blob/develop/botocore/parsers.py#L112. It does not appear to be the case. It only updates based on the kwargs pased in. So it should retain defaults unless you explicitly override them.